Core/PoTItem.cs

```diff
@@ -916,7 +916,12 @@ public void RollAffixes()
 		if (IsUnique)
 		{
 			List<ItemAffix> uniqueItemAffixes = GenerateAffixes();
-			uniqueItemAffixes.ForEach(a => a.Roll());
+			
+			foreach (ItemAffix item in uniqueItemAffixes)
+			{
+				item.Roll();
```

Comment:
Is this just the same code in a different format?

Reply:
I changed this just to make sure there wasn't some weirdness with capturing `item` in the old `ForEach` LINQ call. I don't think that was the issue, but I never reverted that change. LINQ is also not insubstantially slower than this code, not that it matters outside of a hot path.